main.go

```diff
@@ -153,6 +167,7 @@ func createStorageHandle(flags *flagStorage) (storageHandle storage.StorageHandl
 		HttpClientTimeout:   flags.HttpClientTimeout,
 		MaxRetryDuration:    flags.MaxRetryDuration,
 		RetryMultiplier:     flags.RetryMultiplier,
+		UserAgent:           getUserAgent(flags.AppName),
```

Comment:
Your current changes takes care of user agent in Storage client library. We also need to take care when used with jacobsa/gcloud: https://github.com/GoogleCloudPlatform/gcsfuse/blob/5a4355196a0a551505f1175195eafece577a82e4/main.go#L98

Reply:
fixed